src/index.html

```diff
@@ -11,7 +11,192 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="inputs">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            pattern="\w+"
+            autocomplete="off"
+            required
+          >
+        </div>
+
+        <div class="inputs" >
```

Comment:
Please make sure to remove all the spaces
```suggestion
        <div class="inputs">
```